### DISCUSSION TITLE
Camel case format name "dateOptionalTime" is deprecated and will be removed in a future version of Elasticsearch. Use snake case name "date_optional_time" instead.

### DISCUSSION THREAD

**Discussion by adhyayan07** [jertel/elastalert2#627] [Ideas]
![image](https://user-images.githubusercontent.com/87809299/147324606-a209541f-fe47-4125-afe4-ed01895a0476.png)

As of now elastalert-create-index creates the index with old field type format "dateOptionalTime". I updated the mapping format for elastalert2 indices in my instance. 
I don't know from which elasticsearch version they updated this field format, so I have no idea if updating this change in future elastalert2 release will impact old elasticsearch + elastalert2 installations.

**Comment by nsano-rururu** [comment]
We are planning to support elasticsearch 8.

**Reply by nsano-rururu** [reply]
Not an elastalert. elastalert 2.

**Reply by nsano-rururu** [reply]
I also know that the new version of elasticsearch-py will give a warning when creating an index.

**Reply by nsano-rururu** [reply]
https://github.com/jertel/elastalert2/issues/92

**Reply by adhyayan07** [reply]
> Not an elastalert. elastalert 2.

Ah! yes, elastalert2.
